dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/physicalplacement/PhysicalPlacementModel.kt

```diff
@@ -13,35 +13,6 @@
  * ******************************************************************************
  */
 
-package com.spectralogic.dsbrowser.gui.components.physicalplacement;
-
-public class PhysicalPlacementPoolEntryModel {
-    private final String name;
-    private final String health;
-    private final String S3poolType;
-    private final String partition;
-
-    public PhysicalPlacementPoolEntryModel(final String name, final String health, final String S3poolType, final String partition) {
-        this.name = name;
-        this.S3poolType = S3poolType;
-        this.health = health;
-        this.partition = partition;
-    }
-
-    public String getHealth() {
-        return health;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getS3poolType() {
-        return S3poolType;
-    }
-
-    public String getPartition() {
-        return partition;
-    }
-}
+package com.spectralogic.dsbrowser.gui.components.physicalplacement
 
+data class PhysicalPlacementModel(val poolEntries: List<PoolEntry>, val replicationEntries: List<ReplicationEntry>, val tapeEntries: List<TapeEntry>)
```

Comment:
Should these lists be Immutable?

Reply:
Or do they need to be observable collections?

Reply:
Aren't kotlin Lists immutable by default?

Reply:
I believe you are correct, but I prefer the explicitness of Guava's ImmutableList.  That and it's more consistent with the rest of the code base.

Reply:
:+1:  I'll get that switched over